commit 650d00f5f4a1d07d10a9f3dfa7a7028983782b38
Author: Jonathan Riddell <jr@jriddell.org>
Date:   Fri May 6 15:26:02 2016 +0100

    allow distro version number to be overridden in config file
    
    Summary: allow distro version number to be overridden in config file
    
    Test Plan: edit /etc/xdg/kcm-about-distrorc
    
    Reviewers: sitter
    
    Subscribers: plasma-devel
    
    Projects: #plasma
    
    Differential Revision: https://phabricator.kde.org/D1549

diff --git a/Modules/about-distro/src/Module.cpp b/Modules/about-distro/src/Module.cpp
index d7ffefc..fcafb24 100644
--- a/Modules/about-distro/src/Module.cpp
+++ b/Modules/about-distro/src/Module.cpp
@@ -127,7 +127,8 @@ void Module::load()
     // For example OS Ubuntu may be rebranded as Kubuntu. Also Kubuntu Active
     // as a product brand is different from Kubuntu.
     QString distroName = cg.readEntry("Name", os.name);
-    ui->nameVersionLabel->setText(QStringLiteral("%1 %2").arg(distroName, os.versionId));
+    QString versionId = cg.readEntry("Version", os.versionId);
+    ui->nameVersionLabel->setText(QStringLiteral("%1 %2").arg(distroName, versionId));
 
     QString url = cg.readEntry("Website", os.homeUrl);
     if (url.isEmpty())
